generator/.DevConfigs/9fa1d027-8266-430e-b4c0-a541eb733f8e.json

```diff
@@ -0,0 +1,18 @@
+{
+    "core": {
+        "changeLogMessages": [
+            "Updated credential providers to provide Async version for generating credentials."
+        ],
+        "type": "patch",
+        "updateMinimum": true
+    },
+    "services": [
+        {
+            "serviceName": "SecurityToken",
+            "type": "patch",
+            "changeLogMessages": [
+                "Added implementation for ICoreAmazonSTS_SAML/ICoreAmazonSTS.CredentialsFromSAMLAuthenticationAsync() and ICoreAmazonSTS.CredentialsFromAssumeRoleAuthenticationAsync()."
```

Comment:
Make the change log something users will understand which don't directly interact with the `ICoreX` interfaces. Something like add async code paths for the SAML and Assume role credentials providers.

Reply:
Done.